framework/PageSettings/PageSettingsDetails.tsx

```diff
@@ -0,0 +1,67 @@
+import { ButtonVariant } from '@patternfly/react-core';
+import { DropdownPosition } from '@patternfly/react-core/deprecated';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { useContext, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { IPageAction, PageActionSelection, PageActionType } from '../PageActions/PageAction';
+import { PageActions } from '../PageActions/PageActions';
+import { PageDetail } from '../PageDetails/PageDetail';
+import { PageDetails } from '../PageDetails/PageDetails';
+import { PageHeader } from '../PageHeader';
+import { PageLayout } from '../PageLayout';
+import { PageSettingsContext } from './PageSettingsProvider';
+import { IPageSettingsOption, usePageSettingsOptions } from './usePageSettingOptions';
+
+export function PageSettingsDetails() {
+  const { t } = useTranslation();
+  const [settings] = useContext(PageSettingsContext);
+  const navigate = useNavigate();
+  const actions = useMemo<IPageAction<object>[]>(
+    () => [
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.None,
+        variant: ButtonVariant.primary,
+        icon: PencilAltIcon,
+        label: t('Edit'),
+        onClick: () => navigate('./edit'),
+        isPinned: true,
+      },
+    ],
+    [navigate, t]
+  );
+  const options = usePageSettingsOptions();
+
+  return (
+    <PageLayout>
+      <PageHeader
+        title={t('User Preferences')}
```

Comment:
```suggestion
        title={t('User preferences')}
```
Based on recent changes from @tiyiprh we probably want this to me sentence case.

Reply:
Since this is a page title, it should be title case matching the navigation. If it were a create page it would be sentence case. Sorry there are slight differences that could be confusing.